.circleci/config.yml

```diff
@@ -216,40 +213,27 @@ jobs:
   docker-compose-test:
     executor: ubuntu-medium
     steps:
-      - checkout
-      - aws-setup
-      - aws-ecr-setup
-      - local-deploy-setup
-      - run: echo "export IMAGE_TAG=dev_$(git rev-parse --short=8 HEAD)" >> $BASH_ENV
-      # since we're running with `--build-all`, assume that if it passes, we have all images required for Forge
-      - run: aws ecr describe-images --repository-name="aptos/validator" --image-ids=imageTag=$IMAGE_TAG
-      - run:
-          name: Start local testnet docker-compose
-          shell: /bin/bash
-          working_directory: docker/compose/validator-testnet
-          command: |
-            export VALIDATOR_IMAGE_REPO=${AWS_ECR_ACCOUNT_URL}/aptos/validator
-            export FAUCET_IMAGE_REPO=${AWS_ECR_ACCOUNT_URL}/aptos/faucet
-            docker-compose up -d
-      - run:
-          name: Verify testnet docker-compose up
-          shell: /bin/bash
-          command: |
-            for i in $(seq 60); do
-              curl -s -w "\n%{http_code}\n" localhost:8080 | tee >(tail -1 > validator_ret.txt)
-              curl -s -w "\n%{http_code}\n" -X POST 'localhost:8000/mint?pub_key=459c77a38803bd53f3adee52703810e3a74fd7c46952c497e75afb0a7932586d&amount=20000000' \
-                | tee >(tail -1 > faucet_ret.txt)
-              if [ "$(cat validator_ret.txt)" = "200" ] && [ "$(cat faucet_ret.txt)" = "200" ]; then
-                echo "Both validator and faucet healthy"
-                exit 0
-              fi
-              sleep 1
-            done
-            exit 1
+      - docker-compose-setup
   sdk-typescript-test:
-    executor: docker-typescript-builder
+    executor: ubuntu-medium
     steps:
-      - checkout
+      - docker-compose-setup
+      - dev-setup
+      - run:
+          name: Install Node + Yarn
+          command: |
+            export NODE_VERSION=16.14.2
+            curl -L -o node.tar.xz "https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.xz" && \
+              sudo tar -xJf node.tar.xz -C /usr/local --strip-components=1 && \
+              rm node.tar.xz && \
+              sudo ln -s /usr/local/bin/node /usr/local/bin/nodejs
+            export PATH=~/.yarn/bin:$PATH
+            export YARN_VERSION=1.22.17
+            RUN curl -L -o yarn.tar.gz "https://yarnpkg.com/downloads/${YARN_VERSION}/yarn-v${YARN_VERSION}.tar.gz" && \
```

Comment:
Get rid of the `RUN` ? You can also probably just run each command on its own line without the `&& \`